app/templates/register_login.html

```diff
@@ -27,5 +27,24 @@
         <input type="submit" class="btn-primary u-pull-right animated" value="{{ type }}">
       </div>
     </form>
+    <div class="frame__body form-section">
+      <label>Or {{ type|lower }} with:</label>
+      <div class="row">
+        <div class="col"> 
+          <a href="{% url "social:begin" "google-oauth2" %}?next={{ next }}" alt="{{ type }} with Google" title="Google">
+            <button class="btn-grey u-no-margin w-100">
+              <i class="zmdi zmdi-google"></i>&nbsp;<span>Google</span>
+            </button>
+          </a>
```

Comment:
```suggestion
          <a href="{% url "social:begin" "google-oauth2" %}?next={{ next }}" class="btn-grey u-no-margin w-100">
              <i class="zmdi zmdi-google"></i> Google
          </a>
```
I'm pretty sure this looks the same (maybe not the `<span>` removal), `alt` doesn't get used on `<a>`, and `title` doesn't serve any better purpose here.